include/cv/classification.hpp

```diff
@@ -32,6 +39,11 @@ class Classification {
     ClassificationResults classify(cv::Mat croppedImage, cv::Mat shapeMask, cv::Mat characterMask);
 
  private:
+    torch::jit::script::Module shapeModel;
+    torch::jit::script::Module characterModel;
+    torch::jit::script::Module colorModel;
```

Comment:
I think you can remove the color Model private member here for now. Technically it's not a Pytorch model so we don't need to declare it with that type.